plugin.video.fsgo/default.py

```diff
@@ -50,7 +52,7 @@ def addon_log(string):
         xbmc.log(msg=msg, level=xbmc.LOGDEBUG)
```

Comment:
xbmc.LOGDEBUG should already make sure that log info only gets printed for debug mode. Why do you check for debug mode  via JSON?